src/overlay/FlowControlCapacity.h

```diff
@@ -29,6 +29,7 @@ class FlowControlCapacity
     // Capacity of a connected peer
     uint64_t mOutboundCapacity{0};
     NodeID const& mNodeID;
+    std::optional<uint32_t> mOverlayVersion;
```

Comment:
Nit: Rename to something like `mRemoteOverlayVersion`. This name doesn't make it clear that this value is different than `mApp.getConfig().OVERLAY_PROTOCOL_VERSION`.